src/popup/components/TotalWalletAmount.vue

```diff
@@ -10,20 +10,32 @@
 </template>
 
 <script lang="ts">
-import { computed, defineComponent } from 'vue';
+import {
+  computed,
+  defineComponent,
+  PropType,
+} from 'vue';
 import { useStore } from 'vuex';
-import { useCurrencies } from '../../composables';
+import { Protocol } from '@/types';
+import { PROTOCOL_AETERNITY } from '@/constants';
+import { useCurrencies } from '@/composables';
 
 export default defineComponent({
   props: {
-    totalBalance: { type: String, required: true },
+    totalBalance: { type: String, default: '0' },
+    totalBalanceCurrency: { type: String, default: '' },
```

Comment:
This looks rather bad. In my opinion this components should just receive the balance and display it. Why two separate props?